charts/tenant-base/chart/values.yaml

```
@@ -0,0 +1,72 @@
+image:
+  repository: nginx
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+
+replicaCount: 1
+
+livenessProbe:
+  httpGet:
+    path: /
+    port: http
+
+readinessProbe:
+  httpGet:
+    path: /
+    port: http
```

Make it work with podinfo, and we can't listen on port 80 anyways as non-root:
```suggestion
livenessProbe:
  httpGet:
    path: /healthz
    port: 9898

readinessProbe:
  httpGet:
    path: /healthz
    port: 9898
```